docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -19,3 +19,59 @@ The network parameters between two clusters are exchanged using the`networkconfi
 2. `NCFG 1->2` is replicated to the remote cluster by the `CRDReplicator`, running in Cluster1 who knows how to interact with the API server of Cluster2. Same step done in Cluster2;
 3. In Cluster2 the custom resource `NCFG (1->2)` is processed and its status updated with the NAT information if any. The same is done with `NCFG (2->1)` in Cluster1.
 4. The CRDReplicator running in Cluster1 reflects this changes in the local resource `NCFG(1->2)`. Same step done in Cluster2; The TunnelEndpointCreator in Cluster1 combining the status of `NCFG(1->2)` and the spec of `NCFG (2->1)` it has all the necessary information to create the custom resource `TEP (1-2)` of type `tunnelendpoints.net.liqo.io`.
```

Comment:
The reader here has no idea what NCFG contains (the same for TEP). Please make an example, so that the reader understand the flow better.
Important: if not needed, do not report here the entire structure of a NCFG/TEP, but report here only the most significant information in order to give relevant info to the reader, without distracting it with non-necessary details.